### ISSUE TITLE
Example endpoint not found

### ISSUE THREAD

**Issue by cpup22** [matt-kruse/alexa-app#93]
If I don't include an express route explicitly, for example, the code below works:

```
app.post('/appname', function(req, res) {
    res.json({
      "version": "1.0",
      "response": {
        "shouldEndSession": true,
        "outputSpeech": {
          "type": "SSML",
          "ssml": "<speak>Hmm <break time=\"1s\"/> not sure why we misconfigured this skill</speak>"
        }
      }
    });

});
```

then my node app throws a "There was an error calling the remote endpoint, which returned HTTP 404 : Not Found".  I'm using the example code and it doesn't set the endpoint anywhere.  What am i doing wrong?  (This is more of a clarification than an issue... has to be something I'm doing wrong, although I've literally copied and pasted the example code)

**Comment by dblock** [comment]
@cpup22 What did you do to resolve this? (for the next person)

**Comment by cpup22** [comment]
There are three entries:
1. alexaApp.express(app, "/echo/"); --in code
2. var alexaApp = new alexa.app("skillname"); --in code
3.  https://example.com/echo/skillname --this is configured inside the amazon skill console

I wanted it to be https://example.com/skillname to invoke my skill so I was confused why my setup wasn't working. Then I noticed the "/echo" within the alexa-app code. as soon as I adjusted entry #3 above to include /echo/skillname instead of just /skillname it worked.
